CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Added
 
+- inf-cdk-python-aws, provide first iteration of the AWS CDK Quickstarter ([#956](https://github.com/opendevstack/ods-quickstarters/pull/956))
```

Comment:
just wondering about the naming:
There's also a terraform cdk - how would the name of that be
inf-terraform-cdk-python?

Based on that, this one should be 
inf-aws-cdk-python :-D

Reply:
actually someone else requested to remove "aws" because its evident ... but I am open to add that again :)

Reply:
as mentioned, there's also terraform cdk so it's not evident, imho :-D